_data/glossary/application-developer.yaml

```diff
@@ -0,0 +1,10 @@
+id: application-developer
+name: Application Developer
+related:
+  - application-architect
+tags:
+  - user-type
+short-description: |
+  A person who writes an application that runs in a Kubernetes cluster.
+long-description: |
+  Typically an application developer works on one portion of an application, which might be monolithic or composed of many microservices.
```

Comment:
An application developer focuses on one part of an application. The scale of their focus may vary significantly in size.